Automattic/jetpack
# Add interstitial for Stats

## Proposed changes:

* This PR adds a product interstitial for Stats in My Jetpack

<img width="1096" alt="image" src="https://github.com/Automattic/jetpack/assets/5714212/9c57a30f-5979-49fb-adcf-444f1180e4ed">

### Other information:


## Jetpack product discussion


pbNhbs-9bI-p2

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Spin up a JN site with Jetpack Beta pointing to this branch
* Visit `/wp-admin/admin.php?page=my-jetpack#/add-stats`
* Confirm the new interstitial matches the screenshot above, and the CTA works
* Test the CTA on the My Jetpack stats card - without a plan it should say "learn more". With a free or tiered stats plan, it should say "Upgrade". With a PWYW or legacy commercial plan, it should say "View"
